DOCS-7402 dedicated info (#201)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-manual.adoc

|===

[NOTE]
====
To install a specific version of Runtime Fabric, transfer the installation package to this path on the controller VM acting as the leader during installation: `/opt/anypoint/runtimefabric/installer.tar.gz`. This will skip downloading the latest package from Anypoint control plane.

In Runtime Fabric, the inbound load balancer runs on each controller node in either shared or dedicated mode: 

*Shared Mode* is the default setting and specifies the internal load balancer runs with the amount of CPU cores and memory you specify.

*Dedicated Mode* specifies that the entire node is available, so you cannot specify the amount of CPU cores and memory.

Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
====

> 

Reviewer: Refer to comments left in "install-aws" page.

Author: Updated this - please see if it's what you intended - thank you!

> Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.

Reviewer 2: ditto prev edits

sudo /opt/anypoint/runtimefabric/init.sh
----
+

> . If the installation process encounters an error, it exits with a message to help indicate how to resolve the error. In most cases, errors can be resolved by updating one or more variables to a valid value, and re-running the `init.sh` script.

Reviewer 2: GLOBAL
```suggestion
. If the installation process encounters an error, it exits with a message to help indicate how to resolve the error.
In most cases, you can resolve errors by updating one or more variables to a valid value and rerunning the `init.sh` script.
```

+
[NOTE]
====
This step will install Runtime Fabric across all VMs to form a cluster. It may take 15-25 minutes or longer to complete.

When the installation completes successfully, the file `/opt/anypoint/runtimefabric/.state/init-complete` is created.

> The internal load balancer is distributed across the controller nodes for shared mode, and is deployed on the internal load balancer nodes for dedicated mode.

Reviewer 2: ```suggestion
The internal load balancer is distributed across the controller nodes for shared mode and is deployed on the internal load balancer nodes for dedicated mode.
```
